aasemble/django/apps/mirrorsvc/models.py

```diff
@@ -125,6 +125,7 @@ class Snapshot(models.Model):
     uuid = models.UUIDField(unique=True, default=uuid.uuid4, editable=False)
     timestamp = models.DateTimeField(auto_now_add=True)
     mirrorset = models.ForeignKey(MirrorSet)
+    tag = models.CharField(max_length=200)
```

Comment:
@sorenh ok, but if i create a seperate model, with tag and snapshot as foreign key, dont i need to create another api-endpoint to fill in the tag